src/server/scripts/Northrend/Ulduar/HallsOfStone/brann_bronzebeard.cpp

```diff
@@ -881,7 +1150,25 @@ class spell_hos_dark_matter_aura : public AuraScript
 
     void Register() override
     {
-        OnEffectRemove += AuraEffectRemoveFn(spell_hos_dark_matter_aura::HandleEffectRemove, EFFECT_0, SPELL_AURA_DUMMY, AURA_EFFECT_HANDLE_REAL);
+        OnEffectRemove += AuraEffectRemoveFn(spell_hos_dark_matter::HandleEffectRemove, EFFECT_0, SPELL_AURA_DUMMY, AURA_EFFECT_HANDLE_REAL);
+    }
+};
+
+class spell_hos_dark_matter_size : public SpellScript
+{
+    PrepareSpellScript(spell_hos_dark_matter_size);
+
+    void HandleApplyTouch()
+    {
+        if (Unit* target = GetHitUnit())
+        {
```

Comment:
No brackets on single line ifs

Reply:
Is this also fine?

if (var)
    me->DoC();
else
{
    me->DoA();
    me->DoB();
}

Or if there’s an "else", should it be like this?
if (var)
{
     me->DoC();
}
else
{
    me->DoA();
    me->DoB();
}

(Sorry for the stupid question.)

Reply:
first version would be preferred iirc